src/swap/changelly.js

```diff
@@ -94,18 +94,17 @@ async function getAddress (
     : addressInfo.publicAddress
 }
 
-function checkReply (reply: Object, request?: EdgeSwapRequest) {
```

Comment:
I would have just made the `request` mandatory - it has these two parameters inside it, plus other things other error types might need in the future. If Changelly ever has region restrictions, you will need `request` for the `SwapPermissionError` constructor, for example.

Reply:
ok, I will make that change